src/khoj/interface/web/index.html

```diff
@@ -57,6 +57,27 @@
             }).join("\n") + `</div>`;
         }
 
+        function render_mutliple(query, data, type) {
```

Comment:
Nice! 👌🏾 this should help rendering results from the parallel search across multiple content types in #232 🌈